### PR TITLE
Added pitfall section with logging configuration problem

### PR DESCRIPTION
**Pull request by michael-schnell**
See https://github.com/quarkusio/quarkus/issues/2491 for discussion

### CODE REVIEW

File: docs/src/main/asciidoc/logging.adoc [hunk 1]
```diff
@@ -198,6 +198,39 @@ Applications and components may use any of the following APIs for logging, and t
...
+
+This is usually because you have some agent or something which has initialized logging too early. If that's the case, the workaround is to use the following settings.
+
+For unit tests in your IDE:
+[source]
+----
+-Djava.util.logging.manager=org.jboss.logmanager.LogManager
+----
+
+For your Maven configuration:
```
Comment (machi1990) [inline docs/src/main/asciidoc/logging.adoc]: Duplicate of the section just below - https://github.com/quarkusio/quarkus/blob/544773f4c97cbebd0e055d069970fc3e519a9a4a/docs/src/main/asciidoc/logging.adoc#how-to-configure-logging-for-quarkustest ?

Comment (michael-schnell) [reply docs/src/main/asciidoc/logging.adoc]: I integrated it now to make it more clear.

**Comment by dmlloyd** [comment]
I'm not really familiar with the docs anymore so I don't think I have much of use to add.

**Comment by michael-schnell** [comment]
Reverted the change - Will do it later

**Pull request closed without merging**